px-map-behavior-vector-layer.es6.js

```diff
@@ -0,0 +1,147 @@
+(function() {
+  'use strict';
+
+  /****************************************************************************
+   * BEHAVIORS
+   ****************************************************************************/
+
+  /* Ensures the behavior namespace is created */
+  window.PxMapBehavior = (window.PxMapBehavior || {});
+
+  /**
+   *
+   * @polymerBehavior PxMapBehavior.VectorLayer
+   */
+  PxMapBehavior.VectorLayerImpl = {
+    properties: {
+      /**
+       * An object that will contain the geojson data to display:
+       *
+       * @type {Object}
+       */
+      data: {
+        type: Object,
+        observer: 'shouldUpdateInst',
+        reflectToAttribute: true
```

Comment:
Avoid using `reflectToAttribute` on any properties unless its absolutely necessary to do so (usually only if you want to style the component with CSS using that reflected property). There is a large performance hit for Polymer when dealing with sets/updates to reflected properties, especially on arrays/objects.